packages/ui/Touchable/Touchable.js

```diff
@@ -501,18 +501,10 @@ const Touchable = hoc(defaultConfig, (config, Wrapped) => {
 				this.setState(activate);
 			}
 
-			if (ev && 'changedTouches' in ev) {
-				this.target.focus();
-			}
-
 			return true;
 		}
 
-		deactivate (ev) {
-			if (ev && 'changedTouches' in ev) {
-				this.blurJob.start(this.target);
```

Comment:
Is `this.blurJob` used elsewhere or should we clean it up as well?

Reply:
Looks like this it's only job (:rimshot:) so let's clean up it and the `Job` import too.

Reply:
removed!